packages/design-system-docs/src/pages/components/UsaBanner/usabanner.example.html

```diff
@@ -0,0 +1,47 @@
+<div class="ds-c-usa-banner ds-u-padding-x--3">
```

Comment:
We generally dont want to use utilities with the banner classes so its easier for folks to override